keanu-project/src/test/java/io/improbable/keanu/algorithms/mcmc/initialconditions/MultimodalSimulatedAnnealingTest.java

```diff
@@ -70,7 +70,7 @@ public void findsModesForDiscreteContinuousHybridNetwork() {
             .then(B)
             .orElse(D);
 
-        DoubleVertex G = new GaussianVertex(new CastDoubleVertex(F), 1.5);
+        DoubleVertex G = new GaussianVertex(new CastToDoubleVertex(F), 1.5);
```

Comment:
F is already a DoubleVertex. The cast is no longer need and probably there from a previous refactor where F was a Vertex<DoubleTensor> before.